### PR TITLE
Moved favicon link and added site meta description

### PR DESCRIPTION
**Pull request by agundy**
also autoformatted the index.html file

### CODE REVIEW

File: client/app/achievements/achievements.html [hunk 1]
```diff
@@ -11,34 +11,11 @@
 
 <div class="container">
   <div class="row text-area">
-    <div class="col-md-8">
```
Comment (kmcnellis) [inline client/app/achievements/achievements.html]: Looks like you have a problem here

Comment (agundy) [inline client/app/achievements/achievements.html]: nope intentional. have to swap the html elements to put the sidebar on top

Comment (kmcnellis) [inline client/app/achievements/achievements.html]: This change shouldn't cause such a major change to existing code

Comment (agundy) [inline client/app/achievements/achievements.html]: this isn't for the image fix. this is another fix put in too

File: client/app/main/main.html [hunk 2]
```diff
@@ -11,35 +11,7 @@
 
 <div class="container">
   <div class="row text-area">
-    <div class="col-md-8">
```
Comment (kmcnellis) [inline client/app/main/main.html]: Same here. A major indentation refactor should prolly be in a separate PR

File: client/app/main/main.html [hunk 3]
```diff
@@ -11,35 +11,7 @@
...
-      At the beginning of each semester, the Internal Advisory Board will
-      review project proposals and decide which projects should receive
-      financial support.
-      </p>
-      <h2>Donations</h2>
-      RCOS uses funds to financially support open source software development. Follow this <a href="https://securelb.imodules.com/s/1225/giving/index.aspx?sid=1225&gid=1&pgid=1947&cid=4524&appealcode=WGAFP&_ga=1.191668650.1206279174.1408978267" title="RPI Donate Link">link</a> if you can help us out by donating!
-    </div>
-
-    <div class="col-md-4 statistics">
+    <div class="col-md-4 col-md-push-8 statistics">
```
Comment (kmcnellis) [inline client/app/main/main.html]: Why call the class 'statistics'?

File: client/app/achievements/achievements.html (hunk 1 above)
Comment (agundy) [inline client/app/achievements/achievements.html]: moves the sidebar to the top of the page on mobilescreenshot 2016-09-27 at 12 31 33 pm

File: client/index.html [hunk 4]
```diff
@@ -3,125 +3,126 @@
 <!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
 <!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
 <!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
-  <head>
```
Comment (kmcnellis) [inline client/index.html]: Same here

Comment (kmcnellis) [inline client/index.html]: It is hard to see actual changes

File: client/index.html [hunk 5]
```diff
@@ -3,125 +3,126 @@
...
-      <script src="components/sortArrow/sortArrow.directive.js"></script>
-      <script src="app/account/settings/confirmDelete/confirmDelete.directive.js"></script>
-      <script src="components/util/util.service.js"></script>
-      <!-- endinjector -->
-    <!-- endbuild -->
-  </body>
+  <![endif]-->
+  <!-- build:js({client,node_modules}) app/vendor.js -->
+  <!-- bower:js -->
+  <script src="bower_components/jquery/dist/jquery.js"></script>
```
Comment (kmcnellis) [inline client/index.html]: This refactor will cause many merge conflicts

File: client/app/main/main.html (hunk 2 above)
Comment (agundy) [inline client/app/main/main.html]: this isn't a indentation refactor. just moved the element. It's fine in this commit.

File: client/app/main/main.html (hunk 3 above)
Comment (agundy) [inline client/app/main/main.html]: because this is the statistics column on the main page?

File: client/index.html (hunk 5 above)
Comment (agundy) [inline client/index.html]: this file is autogenerated. it should be a super easy rebase

File: client/app/achievements/achievements.html (hunk 1 above)
Comment (kmcnellis) [inline client/app/achievements/achievements.html]: OK

Comment (kmcnellis) [inline client/app/achievements/achievements.html]: You could also do this using the .col-xs-pull-#  class. https://scotch.io/tutorials/reorder-css-columns-using-bootstrap

Comment (kmcnellis) [inline client/app/achievements/achievements.html]: Can't you push & pull without re-ordering the code?

Comment (kmcnellis) [inline client/app/achievements/achievements.html]: Just add a .col-xs-pull / push class?

Comment (agundy) [inline client/app/achievements/achievements.html]: oh nice. Just read from here:  http://stackoverflow.com/questions/20171408/how-do-i-change-bootstrap-3-column-order-on-mobile-layout `You cannot change the order of columns in smaller screens but you can do that in large screens` guess it's changed.

File: client/app/main/main.html (hunk 3 above)
Comment (kmcnellis) [inline client/app/main/main.html]: Its also the sidebar & meeting times column

File: client/app/achievements/achievements.html [hunk 6]
```diff
@@ -11,34 +11,11 @@
...
-            <div class="panel-body">
-              <div id="content" class="event-description" btf-markdown="ach.description"></div>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-
-    <div class="col-md-4 statistics">
+    <div class="col-md-4 col-md-push-8 statistics">
```
Comment (kmcnellis) [inline client/app/achievements/achievements.html]: And this one on the achievements page doesn't even have statistics in it

File: client/app/achievements/achievements.html (hunk 1 above)
Comment (kmcnellis) [inline client/app/achievements/achievements.html]: I think we might not want statistics to be the first thing a mobile user sees - I like seeing the mission statement and such. It describes what we do.

File: client/app/main/main.html (hunk 2 above)
Comment (kmcnellis) [inline client/app/main/main.html]: ok

File: client/index.html (hunk 5 above)
Comment (kmcnellis) [inline client/index.html]: But why do it at all?

File: client/index.html (hunk 4 above)
Comment (kmcnellis) [inline client/index.html]: And I liked the tabs. It made it clearer to read 😞

File: client/app/achievements/achievements.html (hunk 6 above)
Comment (agundy) [inline client/app/achievements/achievements.html]: will fix

File: client/app/main/main.html (hunk 3 above)
Comment (agundy) [inline client/app/main/main.html]: will change the classname

File: client/index.html (hunk 4 above)
Comment (agundy) [inline client/index.html]: it didn't meet our coding style guide..

File: client/index.html (hunk 5 above)
Comment (agundy) [inline client/index.html]: match our style guide

Comment (kmcnellis) [inline client/index.html]: Our style guide?

File: client/app/achievements/achievements.html (hunk 1 above)
Comment (agundy) [inline client/app/achievements/achievements.html]: wanted to bump the navigation to the top of the page

File: client/index.html (hunk 4 above)
Comment (kmcnellis) [inline client/index.html]: Our style guide only says:
'''## HTML Files
- Double quotes over single quotes for attributes
- Comment at closing of element where it closes via class i.e.
```HTML
<div class="row">
  ...
</div><!-- end: row -->
```
'''

Comment (kmcnellis) [inline client/index.html]: Nothing about tabs

Comment (kmcnellis) [inline client/index.html]: This file I would also consider a grunt - style file. The comments are not for usability, but actually dictate sections of the file.

Comment (agundy) [inline client/index.html]: it didn't match the code style guide

File: client/app/achievements/achievements.html (hunk 1 above)
Comment (kmcnellis) [inline client/app/achievements/achievements.html]: Can we just bump nav without the rest? It is probably not the most important thing to users just looking at our site for the first time.

File: client/index.html (hunk 4 above)
Comment (kmcnellis) [inline client/index.html]: Please specify the section to which you are referring. I cannot find anything about making a file completely flat.

**Comment by kmcnellis** [comment]
Just the navbar vs entire statistics thing and the class name now

File: client/index.html (hunk 4 above)
Comment (agundy) [inline client/index.html]: i guess it's just the editor config and this was how my editor autoformatted

Comment (kmcnellis) [inline client/index.html]: Yup. It is :)

**Comment by agundy** [comment]
how do you think it should look/work?

**Comment by kmcnellis** [comment]
Not really sure. But for a new user just finding our site, they should see the part about who we are first. Maybe use tabs instead of a menu?

**Comment by agundy** [comment]
they see the description: `A community of Open Source developers at RPI` maybe we table this discussion for #453 ?
